Vizzuality/marxan-cloud
# Feature/new project

![image](https://user-images.githubusercontent.com/545342/110688331-c05ee000-81e1-11eb-8246-0047c6fa2b70.png)

## Create new project

### Overview

This PR adds all the interface elements plus logic necessary for a first iteration of the "New project" user flow.

### Designs

- https://www.figma.com/file/hq0BZNB9fzyFSbEUgQIHdK/Marxan-Visual_V02?node-id=1530%3A87

### Testing instructions

Go to `http://localhost:3000/projects/new` and check all the different options that are part of the form.

### Feature relevant tickets

- https://vizzuality.atlassian.net/browse/MARXAN-102?atlOrigin=eyJpIjoiMTcyNmNlYjZlZjFjNDBiNWI4ZDIxNWMyZDc2Yzc1MzkiLCJwIjoiaiJ9
